### PR TITLE
087release update

### PR DESCRIPTION
**Pull request by rkooo567**
## Why are these changes needed?

Release update committed to 0.8.7 release branch, but not to the master. 

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/latest/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failure rates at https://ray-travis-tracker.herokuapp.com/.
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested (please justify below)

### CODE REVIEW

**Review by simon-mo — APPROVED** [review]

File: doc/dev/bin/ray-project/cluster.yaml [hunk 1]
```diff
@@ -23,13 +23,16 @@ head_setup_commands:
     - wget --quiet https://repo.continuum.io/archive/Anaconda3-5.0.1-Linux-x86_64.sh || true
     - bash Anaconda3-5.0.1-Linux-x86_64.sh -b -p $HOME/anaconda3 || true
     - echo 'export PATH="$HOME/anaconda3/bin:$PATH"' >> ~/.bashrc
+    - echo 'RAY_VERSION=0.8.7' >> ~/.bashrc
```
Comment (rkooo567) [inline doc/dev/bin/ray-project/cluster.yaml:26]: @barakmich This value should be changed manually.

File: doc/dev/download_wheels.sh [hunk 2]
```diff
@@ -11,12 +11,10 @@ if [[ -z "$RAY_VERSION" ]]; then
 fi
 
 # Make sure Linux wheels are downloadable without errors.
-wget "https://s3-us-west-2.amazonaws.com/ray-wheels/releases/$RAY_VERSION/$RAY_HASH/ray-$RAY_VERSION-cp35-cp35m-manylinux1_x86_64.whl"
 wget "https://s3-us-west-2.amazonaws.com/ray-wheels/releases/$RAY_VERSION/$RAY_HASH/ray-$RAY_VERSION-cp36-cp36m-manylinux1_x86_64.whl"
 wget "https://s3-us-west-2.amazonaws.com/ray-wheels/releases/$RAY_VERSION/$RAY_HASH/ray-$RAY_VERSION-cp37-cp37m-manylinux1_x86_64.whl"
 wget "https://s3-us-west-2.amazonaws.com/ray-wheels/releases/$RAY_VERSION/$RAY_HASH/ray-$RAY_VERSION-cp38-cp38-manylinux1_x86_64.whl"
 # Make sure macos wheels are downloadable without errors.
-wget "https://s3-us-west-2.amazonaws.com/ray-wheels/releases/$RAY_VERSION/$RAY_HASH/ray-$RAY_VERSION-cp35-cp35m-macosx_10_13_intel.whl"
```
Comment (rkooo567) [inline doc/dev/download_wheels.sh:19]: Context: We don't support Python 3.5 anymore.

File: doc/dev/release_logs/0.8.4/microbenchmark.txt [hunk 3]
```diff
@@ -1,7 +1,7 @@
-single client get calls per second 9887.7 +- 1181.06
-single client put calls per second 5061.57 +- 89.58
+single client get calls (Plasma Store) per second 9887.7 +- 1181.06
```
Comment (rkooo567) [inline doc/dev/release_logs/0.8.4/microbenchmark.txt:1]: cc @barakmich Context: This is the performance stats for non-direct call.

File: doc/dev/bin/pip_download_test.sh [hunk 4]
```diff
@@ -41,7 +45,12 @@ do
     printf "\n\n\n"
 
     pip install redis==3.3.2
-    pip install msgpack==0.6.2
+    pip install msgpack==1.0.0
```
Comment (rkooo567) [inline doc/dev/bin/pip_download_test.sh:48]: cc @barakmich When we push the wheel to test repo of pypi, the wheel doesn't contain all dependencies, so we should manually download them here.

Comment (barakmich) [reply doc/dev/bin/pip_download_test.sh:48]: What gives? The wheel should at least declare its dependencies in setup.py, no?

Comment (barakmich) [reply doc/dev/bin/pip_download_test.sh:48]: ^ @rkooo567

Comment (rkooo567) [reply doc/dev/bin/pip_download_test.sh:48]: I am not sure about the exact details. cc @simon-mo

Comment (simon-mo) [reply doc/dev/bin/pip_download_test.sh:48]: This section is about testing ray wheels from a test PyPI instance. The test PyPI instance is a standalone pip package repository for that doesn't contains our deps.

Comment (simon-mo) [reply doc/dev/bin/pip_download_test.sh:48]: More detail in step 8 of doc/dev/release_process.rst

**Pull request merged**